final_prep/mod_zero_hero.rb

```diff
@@ -56,29 +103,64 @@
 #   - luckyNumbers (array)
 #   - address (hash with following key/values: number , street , state, zip)
 
+scary_monster = {
+  'name' => 'Rotten Onion',
+  'smell' => 'tears',
+  'weight' => 12,
+  'citiesDestroyed' => ['New York', 'London', 'Paris'],
+  'luckyNumbers' => [55, 6, 18],
+  'address' => {'number' => 276, 'street' => 'Onion Ave', 'state' => 'CO', 'zip' => 82467}
+}
 
 # Create a new class called SuperHero
 # - Your class should have the following DYNAMIC values
-#   - name 
+#   - name
 #   - super_power
-#   - age 
+#   - age
 # - Your class should have the following STATIC values
 #   - arch_nemesis, assigned to "The Syntax Error"
 #   - power_level = 100
-#   - energy_level = 50 
+#   - energy_level = 50
+
+class SuperHero
+  def initialize(name, super_power, age)
+    @name = name
+    @super_power = super_power
+    @age = age
+    @arch_nemesis = "The Syntax Error"
+    @power_level = 100
+    @energy_level = 50
+  end
 
 # - Create the following class methods
 #   - say_name, should print the hero's name to the terminal
 #   - maximize_energy, should update the energy_level to 1000
 #   - gain_power, should take an argument of a number and INCREASE the power_level by that number
 
+  def say_name(name)
+    p "#{name}"
+  end
+
+  def maximize_energy
+    @energy_level = 1000
+  end
+
+  def gain_power(number)
+    @energy_level += number
+  end
+end
 # - Create 2 instances of your SuperHero class
 
+ruby = SuperHero.new("Ruby Girl", "Super Coding", "25")
+java = SuperHero.new("Java", "Super Alert", "41")
+
 
 # Reflection
 # What parts were most difficult about this exerise?
+# I had a hard time with the function because I haven't reviewed it in a while.
```

Comment:
They look good to me! Just remember when you're having a hard time Google will help.